examples/access-plugin-minimal/go.mod

```diff
@@ -46,14 +45,13 @@ require (
 	go.opentelemetry.io/otel/sdk v1.22.0 // indirect
 	go.opentelemetry.io/otel/trace v1.22.0 // indirect
 	go.opentelemetry.io/proto/otlp v1.1.0 // indirect
-	golang.org/x/crypto v0.31.0 // indirect
+	golang.org/x/crypto v0.35.0 // indirect
```

Comment:
I've touched everything to avoid scanner false positives, even though arguably examples/ don't matter.